src/Components/home/Calendar.jsx

```diff
@@ -25,16 +26,39 @@ const calendarHeader = {
 export function calendarCall(info, successCallback) {
   API.get(`/calendar?start=${info.start}&end=${info.end}`)
     .then((res) => successCallback(res.data))
-    .catch((err) => console.log(err));
+    .catch((err) => console.error(err));
 }
 
 export default function Calendar() {
+
   return (
     <FullCalendar
       defaultView="dayGridMonth"
-      plugins={[dayGridPlugin, timeGridPlugin]}
-      eventSources={[calendarCall, testEvents]}
+      eventTextColor="white"
+      eventBorderColor="rgba(0, 0, 0, 0)"
+      plugins={[dayGridPlugin, timeGridPlugin, googleCalendarPlugin, interactionPlugin]}
+      googleCalendarApiKey='AIzaSyBREtcqxmgY2SbJB5ddscMlKpWrHM0R1JM'
+      events={{googleCalendarId: "q2o6gp4ogd31rls2q7sqvl5nps@group.calendar.google.com"}}
       header={calendarHeader}
+      eventDataTransform={(info) => {
+        info.url = undefined;
```

Comment:
Why are we removing the event URL on transform?

Reply:
Every event comes with a URL to the google calendar event, so clicking would open google calendar.  We definitely don't want that.  This removes the incoming URLs.  Clicking now pops up the description and location.